### ISSUE TITLE
Federated Simulation for Medical Imaging

### ISSUE THREAD

**Issue by 3D-ICE-arXiv** [3D-ICE-arXiv/Papers#10]
**■論文の不明点**
synthesizing novel3D shapes and materialsなどShapeとMaterialを生み出すことが述べられているが
ShapeとMaterialとは何なのかいまいちピンとこない。

**■どんなもの？**
いろんなデータセットがあるが現在ばらばらであり、それらを統合して使用できればとても良い。

We introduce a physics-driven generative approach that consists of two modules:
1) a module that synthesizes 3D cardiac shapes along with their materials,
2) a CT simulator that renders these into realistic CT volumes. 
Both are implemented as learnable neural network modules enabling us to simulate realistic cardiac CTs.

**■先行研究と比べてどこがすごい？**
CTからMesh-basedなCardiac shape生成が行われてきた。
従来研究では、複数のモダリティや異なる施設間のデータ融合に関する研究が行われてきたが、
それらのデータから新たな新しい形状やAnnotationを生成することはできていない。
さらに、これらの方法ではピクセルレベルでのみ動作し、モダリティーごとにTrainingを行う必要があった。

この論文では統合のステップを省き、３D形状とマテリアルを統合させることで
現行のAnnotationを活用したままで、3D ShapeをSynthesizeできるようになった。
※あまりよく理解していません。

**■技術や手法のキモはどこ？**
図１つめに示すように、
normal distributionから作られたLatent vectorを用いて
Material mapを作るためのCNNモデルG(θM)でMaterial mapを作成し
撮影した機械のDistributionを考慮したDevice parameterを用いてCT SimulatorでMaterial mapを修正
Volume mask(=Shape map?)を作るためのCNNモデルG(θS)でVolume maskを作成、
Volume maskとCT Simulatorの結果からLocal CT Volume Enhance mentを行い
Simulated Dataを作成する

図２に示しているのは、
モデルを作成するにあたり、Semi-Supervised Learningを採用。
いくつかのラベル付けされた＆されていないVolume data（D）から
エッセンスを取り出した3D Segmentation model（S）を作成
LocalなデータをCentral Servcerに送付し、Global（Domain Adaptationをしたようなイメージ？） ShapeとMaterialを返す
そのデータを図１のようにCT Simulatorに飲ませてSimulatedデータを作成する

この操作により施設間の誤差を埋めるようなアウトプットを出すことができる。

**■どうやって有効だと検証した？**
３つの異なるCTデータセットを使用して検証
1)CT20 
data was collected from healthy adults at the Shanghai Shugang Hospital, China (おそらくオリジナルデータ)
provided as part of the MM-WHS challenge [17]
2,3)The CT34LC and CT34MC data [15] were collected from congenital heart disease patients, whose ages ranged from 1 month-21 years.

**■議論はある？**
正直内容を完全に理解していないのでよくわからないが、３D限定である点は我々にとって使いにくい。

**■次に読むべき論文は？*
施設間のSynthesizeは大きなテーマである様子。
ここを掘るわけではないが、Domain Adaptation関連の知見を深めるためには以下を読むとよいかも。
引用[5,13,16]

Generative Latent Optimization (GLO) [2]
  First, we pre-train the model using the labeled training subset, 
  and then fine-tune the model in a semi-supervised fashion using the rest of the unlabeled training data.

![image](https://user-images.githubusercontent.com/73802219/98441493-01ff5180-2142-11eb-8548-eb792c38b3fb.png)

![image](https://user-images.githubusercontent.com/73802219/98441928-e21d5d00-2144-11eb-9774-67bd15cdf3b4.png)
